packages/react-components/src/templates/__tests__/TeamProfileHeader.test.tsx

```diff
@@ -26,6 +26,21 @@ it('renders the name as the top-level heading', () => {
   expect(screen.getByRole('heading').tagName).toBe('H1');
 });
 
+it('renders the tag for inactive teams', () => {
+  render(
+    <TeamProfileHeader
+      {...boilerplateProps}
+      inactiveSince="2022-09-30T09:00:00Z"
+    />,
+  );
+  expect(screen.getByText('Inactive')).toBeVisible();
+});
+
+it('does not render the tag for active teams', () => {
+  render(<TeamProfileHeader {...boilerplateProps} />);
```

Comment:
```suggestion
  render(<TeamProfileHeader {...boilerplateProps} inactiveSince={null} />);
```

Reply:
We try and keep all the bits of the test within the test. Anything in boiler plate props should not be able to change the outcome of this test. This keeps all the test logic within the test